src/geometry/point.cpp

```diff
@@ -11,4 +11,20 @@ double Point::distance(const Point &other) const {
     double zSquared = (z - other.z) * (z - other.z);
 
     return sqrt(xSquared + ySquared + zSquared);
+}
+
+bool Point::operator>(const Point &other) const {
+    return x > other.x && y > other.y && z > other.z;
+}
+
+bool Point::operator<(const Point &other) const {
+    return x < other.x && y < other.y && z < other.z;
+}
```

Comment:
isso aqui tá ambíguo. Pode existir pontos que nem sejam menores ou maiores. Vamos ordenar pelos eixos.
```suggestion

bool Point::operator<(const Point &other) const {
    if(cmp(x, other.x) != 0) return cmp(x, other.x) == -1;
    if(cmp(y, other.y) != 0) return cmp(y, other.y) == -1;
    return cmp(z, other.z) == -1;
}

// geometry/utils.h ??
const double eps = 1e-12;
int cmp(double a, double b){
    if(abs(a - b) < eps)) return 0;
    return (a<b ? -1 : 1);
}
```

Reply:
Feito, renomeei o `geometry/constants` para `geometry/utils` e joguei essa função `cmp` p lá, depois disso atualizei os `operators` do `Point`